### PR TITLE
add gru-npu consent group

### PR DESCRIPTION
**Pull request by chris-s-friedman**

### CODE REVIEW

**Comment by chris-s-friedman** [comment]
looks like CI checks failing on a pip install issue

```
ERROR: Cannot install -r dev-requirements.txt (line 7) and click==7.1.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested click==7.1.2
    black 22.1.0 depends on click>=8.0.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
WARNING: You are using pip version 21.2.4; however, version 22.0.3 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.

Exited with code exit status 1

CircleCI received exit code 1

```

**Comment by fiendish** [comment]
So update click :)

**Comment by chris-s-friedman** [comment]
any ideas on why `test_invalid_ingest_subset_stages` would start failing _now_? the most recent PRs haven't changed the tests _and_ i wouldn't expect this pr to change anything related to this test...

**Comment by fiendish** [comment]
> 2022-02-23T20:05:18.1257895Z E       assert 'invalid choice' in "Usage: test [OPTIONS] INGEST_PACKAGE_PATH\nTry 'test --help' for help.\n\nError: Invalid value for '--stages': 'foo' is not one of 'e', 'et', 'etl', 't', 'tl', 'l'.\n"

> 2022-02-23T20:05:18.1269037Z E       assert 'invalid choice' in "Usage: test [OPTIONS] INGEST_PACKAGE_PATH\nTry 'test --help' for help.\n\nError: Invalid value for '--stages': 'el' is not one of 'e', 'et', 'etl', 't', 'tl', 'l'.\n"

> 2022-02-23T20:05:18.1280658Z E       assert 'invalid choice' in "Usage: test [OPTIONS] INGEST_PACKAGE_PATH\nTry 'test --help' for help.\n\nError: Invalid value for '--stages': 'te' is not one of 'e', 'et', 'etl', 't', 'tl', 'l'.\n"

> 2022-02-23T20:05:18.1292195Z E       assert 'invalid choice' in "Usage: test [OPTIONS] INGEST_PACKAGE_PATH\nTry 'test --help' for help.\n\nError: Invalid value for '--stages': 'lte' is not one of 'e', 'et', 'etl', 't', 'tl', 'l'.\n"

> 2022-02-23T20:05:18.1302738Z E       assert 'invalid choice' in "Usage: test [OPTIONS] INGEST_PACKAGE_PATH\nTry 'test --help' for help.\n\nError: Invalid value for '--stages': 'lt' is not one of 'e', 'et', 'etl', 't', 'tl', 'l'.\n"

**Comment by fiendish** [comment]
Adjust the test to look for "Invalid value" instead of "invalid choice".

**Comment by chris-s-friedman** [comment]
Updated with https://github.com/kids-first/kf-lib-data-ingest/pull/645/commits/0786441fcbeb66ffd009e9a571d3e26a9863d217 . looks like this is the only thing that looked for `invalid choice`.

File: tests/test_ingest_pipeline.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def test_invalid_ingest_subset_stages(
         cli_cmd, [simple_study_cfg, "--stages", stages_to_run_str]
     )
     assert result.exit_code == COMMAND_LINE_ERROR_CODE
-    assert "invalid choice" in result.output
+    assert "invalid value" in result.output
```
Comment (fiendish) [inline tests/test_ingest_pipeline.py:102]: capital Invalid

File: tests/test_ingest_pipeline.py [hunk 2]
```diff
@@ -160,7 +160,9 @@ def test_ingest_no_transform_module(tmpdir):
         os.remove(tm)
 
     # Remove transform module from ingest config
-    ingest_config_path = os.path.join(new_study_dir, "ingest_package_config.py")
+    ingest_config_path = os.path.join(
+        new_study_dir, "ingest_package_config.py"
+    )
```
Comment (fiendish) [inline tests/test_ingest_pipeline.py:165]: Why did this get reshaped?

Comment (fiendish) [reply tests/test_ingest_pipeline.py:165]: anyway, run black -l80 on this file with the new version of black.

Comment (chris-s-friedman) [reply tests/test_ingest_pipeline.py]: it shouldn't have been. Format on save wanted 79 characters instead of 80 🤦 . Updated my settings, and fixed this.

**Review by fiendish — APPROVED** [review]

**Pull request merged**